pennylane/fourier/reconstruct.py

```diff
@@ -0,0 +1,295 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contains a function that computes the fourier series equivalent to
+a quantum expectation value."""
+from functools import wraps
+from inspect import signature
+import warnings
+
+# import numpy as np
+import pennylane as qml
+from pennylane import numpy as np
+
+
+def _reconstruct_equ(fun, num_frequency, fun_at_zero=None):
+    r"""Reconstruct a univariate Fourier series with consecutive integer
+    frequencies, using trigonometric interpolation and equidistant shifts.
+
+    This technique is based on
+    `Dirichlet kernels <https://en.wikipedia.org/wiki/Dirichlet_kernel>`_, see
+    `Vidal and Theis (2018) <https://arxiv.org/abs/1812.06323>`_ or
+    `Wierichs et al. (2021) <https://arxiv.org/abs/2107.12390>`_.
+
+    Args:
+        fun (callable): Function to reconstruct
+        num_frequency (int): Number of integer frequencies in ``fun``
+        fun_at_zero (float): Value of ``fun`` at zero; Providing ``fun_at_zero`` saves one
+            evaluation of ``fun``
+
+    Returns:
+        callable: Reconstructed Fourier series with ``num_frequency`` frequencies,
+        as ``qml.numpy`` based function.
+    """
+    if not abs(int(num_frequency)) == num_frequency:
+        raise ValueError(f"num_frequency must be a non-negative integer, got {num_frequency}")
+
+    a, b = (num_frequency + 0.5) / np.pi, 0.5 / np.pi
```

Comment:
```suggestion
    a = (num_frequency + 0.5) / np.pi
    b = 0.5 / np.pi

```

minor, just for readability!